scripts/conversion_script.py

```diff
@@ -81,6 +81,54 @@ def to_dict(self):
         }
 
 
+def _check_ini_file_modified():
+    rpm_va_output, ini_file_not_modified = run_subprocess(
+        ["/usr/bin/rpm", "-Va", "convert2rhel"]
+    )
+
+    # No modifications at all
+    if not ini_file_not_modified:
+        return False
+
+    lines = rpm_va_output.strip().split("\n")
+    for line in lines:
+        line = line.strip().split()
+        status = line[0].replace(".", "").replace("?", "")
+        path = line[-1]
+
+        default_ini_modified = path == "/etc/convert2rhel.ini"
+        md5_hash_mismatch = "5" in status
+
+        if default_ini_modified and md5_hash_mismatch:
+            return True
+    return False
+
+
+def check_convert2rhel_inhibitors_before_run():
+    """
+    Conditions that must be True in order to run convert2rhel command.
+    """
+    ini_report = (
+        "Custom or modified ini file would cause re-registration of system to Red Hat Insights "
+        "therefore conversion was inhibited. "
+        "Please restore the /etc/convert2rhel.ini to its original state by running "
+        "'yum reinstall convert2rhel' or uninstall convert2rhel by running "
+        "'yum remove convert2rhel' "
+        "(running conversion task again will install convert2rhel again)."
+    )
+    custom_ini_exists = os.path.exists("~/.convert2rhel.ini")
```

Comment:
Maybe we should use https://docs.python.org/2.7/library/os.path.html#os.path.expanduser?

Reply:
Added!